ui/components/institutional/jwt-url-form/jwt-url-form.scss

```diff
@@ -0,0 +1,58 @@
+.jwt-url-form {
+  flex-flow: column;
+
+  &__jwt-container {
+    width: 100%;
+  }
```

Comment:
we can use width={BLOCK_SIZES.FULL} and other other utility props with box to avoid css.

Reply:
Oh, you're 100% right! I'll add this change in my next PR 💪🏼